src/main/antora/modules/archive/pages/jakarta-pages/jakarta-pages.adoc

```diff
@@ -0,0 +1,3 @@
+= Jakarta Pages
+
+[TBD]
```

Comment:
It's ok to add a line which explains 'Jakarta Pages' was previously known as 'JSP' and add links to

- https://docs.oracle.com/javaee/5/tutorial/doc/bnagx.html
- https://docs.oracle.com/javaee/5/tutorial/doc/bnajo.html
- https://docs.oracle.com/javaee/5/tutorial/doc/bnalj.html